test/lib/package-exports.js

```diff
@@ -0,0 +1,38 @@
+import test from "ava";
+import {createRequire} from "node:module";
```

Comment:
should we add a length test for the exported modules? What happens if nomeone adds a new export but forgot to add it here? Test will not fail, isn't?

Reply:
Done. New test was added.